ti/instrumentation/utils.go

```diff
@@ -509,9 +509,9 @@ func DownloadFile(ctx context.Context, path, url string, fs filesystem.FileSyste
 	return nil
 }
 
-func GetV2AgentDownloadLinks(ctx context.Context, config *tiCfg.Cfg) ([]ti.DownloadLink, error) {
+func GetV2AgentDownloadLinks(ctx context.Context, config *tiCfg.Cfg, useQAEnv bool) ([]ti.DownloadLink, error) {
 	c := config.GetClient()
-	links, err := c.DownloadLink(ctx, "RunTestV2", runtime.GOOS, runtime.GOARCH, "", "", "")
+	links, err := c.DownloadLink(ctx, "RunTestV2", runtime.GOOS, runtime.GOARCH, "", "", "qa")
```

Comment:
will we always send "qa" parameter

Reply:
You are entirely right. I actually missed pushing that change (it will address the previous comment as well)